airflow/models/connection.py

```diff
@@ -251,6 +251,7 @@ def get_hook(self):
         elif self.conn_type == 'grpc':
             from airflow.contrib.hooks.grpc_hook import GrpcHook
             return GrpcHook(grpc_conn_id=self.conn_id)
+        raise AirflowException("Hook for connection type {} not found".format(self.conn_type))
```

Comment:
"Not found" isn't quite right - unknown hook type would be closer I think.

Reply:
Thanks, updated.